src/components/shared/basic/Dropzone/index.tsx

```diff
@@ -111,6 +112,17 @@ export const Dropzone = ({
     [currentFiles, onChange, handleDelete]
   )
 
+  const handleFileSizeValidator = (file: File) => {
+    if (!maxFileSize) return null
+    const fileSizeInMB = (file.size / CONVERT_TO_MB).toFixed(2)
+    return file.size > maxFileSize
+      ? {
+          code: 'size-too-large',
+          message: `File is larger than ${fileSizeInMB} MB`,
```

Comment:
translate message with `t('shared.dropzone.error.fileTooLarge')` and add the text in the assets/locale/ files.